### DISCUSSION TITLE
Need directory browse option, How to assign directory

### DISCUSSION THREAD

**Discussion by GowriShanakar** [rejetto/hfs#451] [General]
Need directory browse option, How to assign directory
![dir](https://github.com/rejetto/hfs/assets/90929359/2a0d2b3a-10ff-49f4-9b4c-5cef5ee83153)

**Comment by rejetto** [comment]
<img width="122" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/752f3898-4554-478d-826a-a777eff9eed3">

you entered here the path on your disk, but instead you have to enter the "web" path (without the domain part).
I understand the confusion, and I will make this clearer in next version.
